app/helpers/series_helper.rb

```diff
@@ -1,28 +1,28 @@
-module SeriesHelper 
-  
+module SeriesHelper
+
   def show_series_data(work)
     series_data = series_data_for_work(work)
     series_data.join(ArchiveConfig.DELIMITER_FOR_OUTPUT).html_safe
   end
-  
+
   # this should only show prev and next works visible to the current user
   def series_data_for_work(work)
     series = work.series.select{|s| s.visible?(current_user)}
     series.map do |serial|
       serial_works = serial.serial_works.find(:all, :include => :work, :conditions => ['works.posted = ?', true], :order => :position).select{|sw| sw.work.visible(current_user)}.collect{|sw| sw.work}
-      visible_position = serial_works.index(work) || serial_works.length     
+      visible_position = serial_works.index(work) || serial_works.length
       unless !visible_position
-        previous_link = (visible_position > 0) ? link_to("&laquo; ".html_safe, serial_works[visible_position - 1]) : "".html_safe
-        main_link = ("Part " + (visible_position+1).to_s + " of the " + link_to(serial.title, serial) + " series").html_safe
-        next_link = (visible_position < serial_works.size-1) ? link_to(" &raquo;".html_safe, serial_works[visible_position + 1]) : "".html_safe
+        previous_link = (visible_position > 0) ? link_to("&larr; Previous Work".html_safe, serial_works[visible_position - 1]) : "".html_safe
```

Comment:
The text in the `link_to` should be wrapped in the `ts` method which some day will allow us to track down all translatable strings:

```ruby
previous_link = visible_position > 0 ? link_to(ts("&larr; Previous Work").html_safe, serial_works[visible_position - 1]) : "".html_safe
```